Makefile

```diff
@@ -102,15 +102,15 @@ push-containers: $(CONTAINERS)
 builder:
 	cd tools/builders/launcher-builder/1.11/ && gcloud builds submit --project=kolide-public-containers --config=cloudbuild.yml
 
-binary-bundle: xp-codesign
+binary-bundle: VERSION = $(shell git describe --tags --always --dirty)
+binary-bundle: #xp-codesign
```

Comment:
rm comment

Reply:
Otherway here -- it should be uncommented. Nice catch